pkg/chaosdaemon/tc_server.go

```diff
@@ -405,3 +431,21 @@ func convertTbfToArgs(tbf *pb.Tbf) string {
 
 	return args
 }
+
+func abstractTcFilter(tc *pb.Tc) string {
```

Comment:
Use `filterExist(tc *pb.Tc) bool`, as the `string` is meaningless.

Reply:
This filter key is used to distinguish TC rules with the same filter. If we don't consider this, `filterExist(tc *pb.Tc) bool` is great!

Reply:
OK. I understand.